src/base1/cockpit.js

```diff
@@ -99,7 +99,7 @@ function invoke_functions(functions, self, args) {
 /* -------------------------------------------------------------------------
  * Channels
  *
- * Public: https://files.cockpit-project.org/guide/api-cockpit.html
+ * Public: http://cockpit-project.org/guide/latest/api-cockpit.html
```

Comment:
I agree with the assessment of the api-cockpit link. Since this is a comment for base1, why not link to http://cockpit-project.org/guide/latest/api-base1.html instead?